README.md

```diff
@@ -11,9 +11,9 @@ A Python wrapper for the MOSTLY AI platform (https://app.mostly.ai/).
 | Live probe the generator on demand              | `mostly.probe(g, size \| seed)`  |
 | Connect to any data source within your org      | `mostly.connect(config)`         |
 | Info about the current user                     | `mostly.me()`                    |
-| Info about the usage                            | `mostly.usage_report()`         |
+| Info about the usage                            | `mostly.usage_report()`          |
 | Info about the platform                         | `mostly.about()`                 |
-
+| Info about the available models                 | `mostly.models(model_type)`      |
```

Comment:
maybe add them in the same order as in the api, eg with .about() being the last one (as typical Menus look like)

Reply:
I moved the line above in both PRs.
Not sure if the order in public API really is meaningful tho? The previous rows didn't follow it. And if we follow the order, the last one would be `mostly.usage_report()`.